specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/ipAddressManager.json

```diff
@@ -0,0 +1,1119 @@
+              "$ref": "#/definitions/StaticCidr"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "StaticCidrs_Get": {
+            "$ref": "./examples/StaticCidrs_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "StaticCidrs"
+        ],
+        "summary": "Delete the Static CIDR resource.",
+        "operationId": "StaticCidrs_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "poolName",
+            "description": "Pool resource name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]*$"
+          },
+          {
+            "in": "path",
+            "name": "staticCidrName",
+            "description": "StaticCidr resource name to delete.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]*$"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Static CIDR delete request accepted and will complete asynchronously",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Static CIDR delete request succeeded"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "StaticCidrs_Delete": {
+            "$ref": "./examples/StaticCidrs_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IpType": {
+      "description": "Enumeration to indicate the IP type.",
+      "enum": [
+        "IPv4",
+        "IPv6"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "IpType"
+      }
+    },
+    "IpamPool": {
+      "description": "Instance of Pool resource.",
+      "type": "object",
+      "required": [
+        "properties"
+      ],
```

Comment:
compared to 18922 from private this was added only in this pr

Reply:
Yes same as here: https://github.com/Azure/azure-rest-api-specs/pull/30560#discussion_r1764053823